air/src/utils.rs

```diff
@@ -87,6 +87,32 @@ pub fn u32_to_bits_le<FA: FieldAlgebra>(val: u32) -> [FA; 32] {
     })
 }
 
+/// Convert a 64-bit integer into an array of 64 0 or 1 field elements.
+///
+/// The output array is in little-endian order.
+#[inline]
+pub fn u64_to_bits_le<FA: FieldAlgebra>(val: u64) -> [FA; 64] {
+    // We do this over F::from_canonical_u64 as from_canonical_u64 can be slow
+    // like in the case of monty field.
+    array::from_fn(|i| {
+        if val & (1 << i) != 0 {
+            FA::ONE
+        } else {
+            FA::ZERO
+        }
+    })
```

Comment:
Nit:
```suggestion
    array::from_fn(|i| FA::from_bool(val & (1 << i) != 0))
```
I think `from_bool` should be fast for all fields, so you can also remove the comment above about Monty fields.

Reply:
Currently `from_bool` is implemented in a silly way for `Monty31` fields. I'll fix that here and update this,